python/lambdas/copy-files-from-tdr/lambda_function.py

```diff
@@ -0,0 +1,84 @@
+import os
+import json
+import boto3
+
+s3_client = boto3.client('s3')
+sqs_client = boto3.client('sqs')
+
+
+def lambda_handler(event, context):
+    destination_bucket = os.environ["DESTINATION_BUCKET"]
+    destination_queue = os.environ["DESTINATION_QUEUE"]
+    for record in event['Records']:
+        body = json.loads(record['body'])
+        file_id = body['fileId']
+        source_bucket = body['bucket']
+        file_location = copy_object(destination_bucket, file_id, source_bucket)
+        copy_object(destination_bucket, f"{file_id}.metadata", source_bucket)
+
+        sqs_client.send_message(QueueUrl=destination_queue, MessageBody=json.dumps({'location': file_location}))
```

Comment:
I'm going to leave it as location because it's location in the ingest process now. We may (unlikely) end up with non-s3 locations and we'd have to change it downstream.